query-data/explore.mdx

```diff
@@ -246,4 +246,15 @@ In the event timeline, line charts, and heat maps, you can drag the pointer over
 - **Zoom** enlarges the section of the chart you highlighted.
 - **Show events** displays events in the selected time range in the event details view.
 
-The time range of your query automatically updates to match what you selected.
\ No newline at end of file
+The time range of your query automatically updates to match what you selected.
+
+### Search within query results
+
+To quickly search for an expression and highlight its occurrences within the query results:
+
+1. In the query results view, press <kbd>Cmd/Ctrl</kbd>&nbsp;<kbd>F</kbd>.
+1. Type the expression that you want to search for.
+1. Press <kbd>Enter</kbd> to highlight the matches and jump to the first match.
```

Comment:
The first match should get jumped on automatically, enter/shift enter are only for navigation